ar6/mapping_template_AR6.csv

```diff
@@ -24,28 +24,29 @@ idx;Variable_AR6;Unit;r21m42;r21m42_unit;r21m42_factor;r21m42_spatial;m42;m42_un
+21;Carbon Sequestration|CCS|Biomass|Energy|Supply|Liquids;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Biomass|Supply|Liquids|w/ couple prod;Mt CO2/yr;;;;;;Carbon Management|Storage|Biomass|Pe2Se|+|Liquids w/ couple prod;Mt CO2/yr;;;;;;;;total carbon dioxide emissions captured from bioenergy use in liquid fuel production (part of IPCC category 1A) and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
 22;Carbon Sequestration|CCS|Biomass|Energy|Supply|Other;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Biomass|Supply|Heat|w/ couple prod;Mt CO2/yr;;;;;;;;;;;;;;;total carbon dioxide emissions captured from bioenergy use in other energy supply (part of IPCC category 1A) and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
 23;Carbon Sequestration|CCS|Biomass|Industrial Processes;Mt CO2/yr;;;;;;;;;;;;;;;;;total carbon dioxide emissions captured from industrial processes from biomass (e.g.,cement production,but not from fossil fuel burning) use and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
-24;Carbon Sequestration|CCS|Fossil;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Fossil|Energy|Demand|Industry;Mt CO2/yr;;;;;;;;;;;;;;;total carbon dioxide emissions captured from fossil fuel use and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
-;Carbon Sequestration|CCS|Fossil;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Fossil|Pe2Se;Mt CO2/yr;;;;;;;;;;;;;;;
+24;Carbon Sequestration|CCS|Fossil;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Fossil|Energy|Demand|Industry;Mt CO2/yr;;;;;;Carbon Management|Storage|+|Fossil|Pe2Se;Mt CO2/yr;;;;;;;;total carbon dioxide emissions captured from fossil fuel use and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
+;Carbon Sequestration|CCS|Fossil;Mt CO2/yr;Emi|CO2|Carbon Capture and Storage|Fossil|Pe2Se;Mt CO2/yr;;;;;;Carbon Management|Storage|Industry Energy|+|Fossil;Mt CO2/yr;;;;;;;;
+;Carbon Sequestration|CCS|Fossil|Demand;Mt CO2/yr;;;;;;;;Carbon Management|Carbon Capture|Industry Energy|+|Fossil;Mt CO2/yr;;;;;;;;total carbon dioxide emissions captured and stored in geological deposits (e.g. in depleted oil and gas fields,unmined coal seams,saline aquifers) and the deep ocean,stored amounts should be reported as positive numbers
```

Comment:
Looks like this is a new variable - then it should be #25 right?
Plus it should alse be Carbon Management|Storage|...

Reply:
I have never touched these numbers. Is there an efficient way to increment them all, if that was desired?

Reply:
I have taken it out, as I can't even remember why I put it in.